test/common/network/utility_test.cc

```diff
@@ -530,22 +530,28 @@ TEST(PortRangeListTest, Normal) {
   }
 }
 
-// TODO(ccaraman): Support big-endian. These tests operate under the assumption that the machine
-// byte order is little-endian.
 TEST(AbslUint128, TestByteOrder) {
+#if defined(ABSL_IS_BIG_ENDIAN)
+  auto BSwapIfRequired = [](const absl::uint128& input) {
```

Comment:
* This should be something like `flip_order_for_endianess`, it's a local definition.
* There already is a `Network::Utility::flipOrder` you can use.